spec/country_select_spec.rb

```diff
@@ -143,9 +143,9 @@ class Walrus
         "United Kingdom",
         "Mexico",
         "Australia",
-        "South Korea"
+        "Freedonia"
```

Comment:
@stefanpenner I changed this test since there is actually a country named `South Korea` :smile: I think countries gem is also looking in the alternative names as well in the canonical one.